tests/parametric/test_sampling_span_tags.py

```diff
@@ -0,0 +1,118 @@
+from utils.parametric.spec.trace import SAMPLING_PRIORITY_KEY  # noqa
+from utils.parametric.spec.trace import SAMPLING_RULE_PRIORITY_RATE  # noqa
+from utils.parametric.spec.trace import Span  # noqa
+from utils.parametric.spec.trace import find_span_in_traces  # noqa
+
+
+def _get_parent_and_child_span(test_agent, test_library):
+    with test_library:
+        with test_library.start_span(name="parent", service="webserver") as parent_span:
+            with test_library.start_span(
+                name="child", service="webserver", parent_id=parent_span.span_id
+            ) as child_span:
+                pass
+
+    traces = test_agent.wait_for_num_spans(2, clear=True)
+
+    parent_span = find_span_in_traces(traces, Span(name="parent", service="webserver"))
+    child_span = find_span_in_traces(traces, Span(name="child", service="webserver"))
+    return parent_span, child_span
+
+
+def _assert_sampling_tags(parent_span, child_span, child_dm, parent_dm, parent_priority, parent_rate):
+    if child_dm is not None or "meta" in child_span:
+        assert child_span["meta"].get(SAMPLING_DECISION_MAKER_KEY) == child_dm
+    if parent_dm is not None or "meta" in parent_span:
+        assert parent_span["meta"].get(SAMPLING_DECISION_MAKER_KEY) == parent_dm
+    assert parent_span["metrics"].get(SAMPLING_PRIORITY_KEY) == parent_priority
+    assert parent_span["metrics"].get(SAMPLING_RULE_PRIORITY_RATE) == parent_rate
+
+
+@scenarios.parametric
+class Test_Sampling_Span_Tags:
+    @pytest.mark.parametrize("library_env", [{"DD_TRACE_SAMPLE_RATE": 1}])
+    @bug(library="python", reason="Python sets dm tag on child span")
+    @bug(library="python_http", reason="Python sets dm tag on child span")
+    @bug(library="nodejs", reason="NodeJS does not set priority on parent span")
+    def test_tags_child_dropped_sst001(self, test_agent, test_library):
+        with test_library:
+            with test_library.start_span(name="parent", service="webserver") as parent_span:
+                with test_library.start_span(
+                    name="child", service="webserver", parent_id=parent_span.span_id
+                ) as child_span:
+                    child_span.set_meta(MANUAL_DROP_KEY, None)
+
+        traces = test_agent.wait_for_num_spans(2, clear=True)
+
+        parent_span = find_span_in_traces(traces, Span(name="parent", service="webserver"))
+        child_span = find_span_in_traces(traces, Span(name="child", service="webserver"))
+
+        _assert_sampling_tags(parent_span, child_span, None, "-3", 2, 1)
+
+    @pytest.mark.parametrize("library_env", [{"DD_TRACE_SAMPLE_RATE": 1}])
+    @bug(library="python", reason="Python sets dm tag on child span")
+    @bug(library="python_http", reason="Python sets dm tag on child span")
+    @bug(library="nodejs", reason="NodeJS sets dm tag -3 on parent span")
+    def test_tags_child_kept_sst007(self, test_agent, test_library):
```

Comment:
What does sst007 refer to? Is there a doc that describes these test cases?

Reply:
It's a style that's used throughout the parametric tests. As far as I know it's not referenced by any other system or documentation.